src/m365/spp/commands/contentcenter/contentcenter-list.ts

```diff
@@ -0,0 +1,74 @@
+import { Logger } from '../../../../cli/Logger.js';
+import config from '../../../../config.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { ClientSvcResponse, ClientSvcResponseContents, FormDigestInfo, spo } from '../../../../utils/spo.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import { SiteProperties } from '../../../spo/commands/site/SiteProperties.js';
+import { SPOSitePropertiesEnumerable } from '../../../spo/commands/site/SPOSitePropertiesEnumerable.js';
+import commands from '../../commands.js';
+
+class SppContentCenterListCommand extends SpoCommand {
+  private allSites?: SiteProperties[];
+
+  public get name(): string {
+    return commands.CONTENTCENTER_LIST;
+  }
+
+  public get description(): string {
+    return 'Gets the URLs of the SharePoint Premium content centers';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['Title', 'Url'];
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    try {
+      if (this.verbose) {
+        await logger.logToStderr(`Retrieving list of content centers...`);
+      }
+
+      this.allSites = [];
```

Comment:
Why do we initialize a variable here? Can't we just return the sites from the method 'this.getAllSites()'?

Reply:
Ok I have changed it, I used before the same approach as in `spo site list` command so we can also update it there.